lib/lunchbot_web/router.ex

```diff
@@ -34,6 +34,11 @@ defmodule LunchbotWeb.Router do
     resources "/options", OptionsController
   end
 
+  scope "/", LunchbotWeb do
+    pipe_through [:browser, :require_admin_user]
```

Comment:
Could we pipe through `:require_authenticated_user` instead if we just want our user to be logged in to view their previous orders

Reply:
ahh yup good catch!